dev/breeze/src/airflow_breeze/utils/docker_command_utils.py

```diff
@@ -592,15 +592,16 @@ def fix_ownership_using_docker(quiet: bool = False):
 
 def remove_docker_networks(networks: list[str] | None = None) -> None:
     """
-    Removes specified docker networks. If no networks are specified, it removes all unused networks.
+    Removes specified docker networks. If no networks are specified, it removes all networks created by breeze.
+    Any network with label "com.docker.compose.project=breeze" are removed when no networks are specified.
     Errors are ignored (not even printed in the output), so you can safely call it without checking
     if the networks exist.
 
     :param networks: list of networks to remove
     """
     if networks is None:
         run_command(
-            ["docker", "network", "prune", "-f"],
+            ["docker", "network", "prune", "-f", "-a", "--filter", "label=com.docker.compose.project=breeze"],
```

Comment:
@potiuk , I changed this after you approved the PR.

Reply:
Nice!